spec/rails_helper.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
What the purpose of the line?

Reply:
to set `test` rails context by default if it's not set

Reply:
It will be setted when you will run `rspec`

Reply:
I remove this line - tests was failed. Maybe `rspec` set it to `test`, because exist this line?